docs/examples.md

```diff
@@ -553,6 +553,34 @@ rspec_task:
     Which will create four shards that will theoretically **run tests 4x faster** by equaly splitting all tests between 
     these four shards.
 
+### RSpec and RuboCop Annotations
+
+Cirrus CI natively supports [RSpec](https://rspec.info/) and [RuboCop](https://rubocop.org/) machine-parsable JSON reports.
+
+To get Behaviour-Driven Development test annotations, simply generate a `rspec` artifact from your linter task:
+
+```yaml
```

Comment:
Please add a container instruction for both examples with Ruby

Reply:
I'd also recommend adding `name: RSpec` and `name: RuboCop`

Reply:
See ac443369329753292143b6e7153bc40574661412.

Reply:
Let's also add the annotations to the main example in the `Ruby` section (after `rspec_script: bundle exec rspec`).